### ISSUE TITLE
NameError: uninitialized constant EventMachine::EM_PROTO_TLSv1 

### ISSUE THREAD

**Issue by satori-avasenin** [eventmachine/eventmachine#797]
Hello, I'm installed the latest eventmachine 1.2.5 and getting same error as described in #773:

Steps to reproduce:

```
console$ docker run --rm -ti jruby:9.1 /bin/bash
root@0d1e65e96039:/# gem install eventmachine
Fetching: eventmachine-1.2.5-java.gem (100%)
Successfully installed eventmachine-1.2.5-java
1 gem installed
root@0d1e65e96039:/# gem install faye-websocket
Fetching: websocket-extensions-0.1.2.gem (100%)
Successfully installed websocket-extensions-0.1.2
Fetching: websocket-driver-0.6.5-java.gem (100%)
Successfully installed websocket-driver-0.6.5-java
Fetching: faye-websocket-0.10.7.gem (100%)
Successfully installed faye-websocket-0.10.7
3 gems installed
root@0d1e65e96039:/# irb
irb(main):001:0> require 'eventmachine'
=> true
irb(main):003:0> require 'faye/websocket'
=> false
irb(main):004:0> EM.run { ws = Faye::WebSocket::Client.new("wss://xxx.api.satori.com/v2") }
NameError: uninitialized constant EventMachine::EM_PROTO_TLSv1
	from org/jruby/RubyModule.java:3340:in `const_missing'
	from /usr/local/bundle/gems/eventmachine-1.2.5-java/lib/em/connection.rb:436:in `start_tls'
	from /usr/local/bundle/gems/faye-websocket-0.10.7/lib/faye/websocket/client.rb:65:in `on_connect'
	from /usr/local/bundle/gems/faye-websocket-0.10.7/lib/faye/websocket/client.rb:76:in `connection_completed'
	from /usr/local/bundle/gems/eventmachine-1.2.5-java/lib/eventmachine.rb:1523:in `event_callback'
	from /usr/local/bundle/gems/eventmachine-1.2.5-java/lib/jeventmachine.rb:94:in `eventCallback'
	from /usr/local/bundle/gems/eventmachine-1.2.5-java/lib/jeventmachine.rb:113:in `run_machine'
	from /usr/local/bundle/gems/eventmachine-1.2.5-java/lib/eventmachine.rb:194:in `run'
	from (irb):4:in `<eval>'
	from org/jruby/RubyKernel.java:1000:in `eval'
	from org/jruby/RubyKernel.java:1298:in `loop'
	from org/jruby/RubyKernel.java:1120:in `catch'
	from org/jruby/RubyKernel.java:1120:in `catch'
	from /opt/jruby/bin/irb:13:in `<main>'
```

**Comment by Littlejd1997** [comment]
Bump. same issue

**Comment by sodabrew** [comment]
Resolved by #791
